include/swift/Remote/MetadataReader.h

```diff
@@ -421,7 +421,8 @@ class MetadataReader {
         }
       } else {
         resolved = Reader->resolvePointer(RemoteAddress(remoteAddress), 0);
-        if (!resolved)
+        if (resolved.getSymbol().empty())
```

Comment:
Nit: I'd add braces here. I know *technically* it doesn't need it, because it's still only followed by a single statement, but still.

Reply:
done